solara/website/pages/documentation/examples/visualization/linked_views.py

```diff
@@ -50,11 +50,10 @@ def on_click_trace(click_data):
         fig.add_trace(px.scatter(x=[click_x], y=[click_y], text=["⭐️"]).data[0])
     # make the figure a bit smaller
     fig.update_layout(width=400)
-    with solara.VBox() as main:
-        solara.FigurePlotly(fig, on_click=on_click_trace)
-        solara.Select(label="X-axis", value=x, values=columns, on_value=set_x)
-        solara.Select(label="Y-axis", value=y, values=columns, on_value=set_y)
-    return main
+    solara.Column()
```

Comment:
There is not need to add a column, you get that 'for free' if you have multiple children, like in this example